data-example/NCT001.json

```diff
@@ -0,0 +1,36 @@
+{
+  "long_title" : "A Phase 1 Multiple Ascending Dose Study of DS-3032b, an Oral MDM2 Inhibitor, in Subjects With Advanced Solid Tumors or Lymphomas",
```

Comment:
Are these mock data?

Reply:
This trial is got from public curation platform. I changed nct_id.

Reply:
Sure